### PR TITLE
Doc | Add detail to Attestation Protocol

### PR DESCRIPTION
**Pull request by David-Engel**

### CODE REVIEW

File: doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml [hunk 1]
```diff
@@ -167,7 +167,7 @@ Modified: Data Source=(local);Initial Catalog=AdventureWorks;Integrated Security
             <related type="Article" href="/sql/connect/ado-net/overview-sqlclient-driver">Overview of the SqlClient driver</related>
         </AttachDBFilename>
         <AttestationProtocol>
-            <summary>Gets or sets the value of Attestation Protocol.</summary>
+            <summary>Gets or sets the value of Attestation Protocol.<br /><br />When no value is specified, secure enclaves are disabled on the connection.</summary>
```
Comment (DavoudEshtehari) [inline doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml:170]: I'd suggest adding it inside the `remarks` section.

File: doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml [hunk 2]
```diff
@@ -167,7 +167,7 @@ Modified: Data Source=(local);Initial Catalog=AdventureWorks;Integrated Security
...
             <returns>Returns the attestation protocol.</returns>
```
Comment (DavoudEshtehari) [inline doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml:171]: It should be `value`.
```suggestion
            <value>The attestation protocol in use.</value>
```

**Review by DavoudEshtehari — APPROVED** [review]

File: doc/snippets/Microsoft.Data.SqlClient/SqlConnection.xml [hunk 3]
```diff
@@ -527,7 +527,7 @@ End Module
...
-|Attestation Protocol|N/A|Gets or sets the value of Attestation Protocol.<br /><br />Valid values are:<br />`AAS`<br />`HGS`<br />`None`|
+|Attestation Protocol|N/A|Gets or sets the value of Attestation Protocol.<br /><br />When no value is specified, secure enclaves are disabled on the connection.<br /><br />Valid values are:<br />`AAS`<br />`HGS`<br />`None`|
```
Comment (johnnypham) [inline doc/snippets/Microsoft.Data.SqlClient/SqlConnection.xml:530]:
```suggestion
|Attestation Protocol|NotSpecified|Gets or sets the value of Attestation Protocol.<br /><br />When no value is specified, secure enclaves are disabled on the connection.<br /><br />Valid values are:<br />`AAS`<br />`HGS`<br />`None`|
```

**Comment by johnnypham** [comment]
The published order for [SqlConnectionAttestationProtocol](https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnectionattestationprotocol?view=sqlclient-dotnet-standard-4.0) isn't the same as the order of the enum values, even though the order in SqlConnectionAttestationProtocol.xml is correct. Any idea why?

**Review by JRahnama — APPROVED** [review]

File: doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml (hunk 1 above)
Comment (David-Engel) [reply doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml]: Done

File: doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml (hunk 2 above)
Comment (David-Engel) [reply doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml]: Fixed

**Comment by David-Engel** [comment]
> The published order for [SqlConnectionAttestationProtocol](https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnectionattestationprotocol?view=sqlclient-dotnet-standard-4.0) isn't the same as the order of the enum values, even though the order in SqlConnectionAttestationProtocol.xml is correct. Any idea why?

Looking at other enums, it looks like the API docs sorts them alphabetically, by default.

**Review by DavoudEshtehari — APPROVED** [review]

**Pull request merged**